src/medication-dispense/medication-dispense.resource.tsx

```diff
@@ -71,16 +90,15 @@ export function useSubstitutionReasonValueSet(uuid: string) {
   };
 }
 
-// TODO: should more be stripped out of here when initializing?... ie don't copy over all the display data?
 // TODO: what about the issue with the repetitive reloading?
 export function initiateMedicationDispenseBody(
   medicationRequest: MedicationRequest,
   session: Session,
-  medicationRequestExpirationPeriodInDays: number
+  populateDispenseInformation: boolean
 ): MedicationDispense {
   let medicationDispense: MedicationDispense = {
     resourceType: "MedicationDispense",
-    status: "completed", // might need to change this to appropriate status
+    status: null,
```

Comment:
can we use a status from the object u introduced for this?

Reply:
The issue here is that the status will vary whether we are submitting via the Dispense, Pause, or Close buttons, so instead of explicitly setting it here like we did previously, we set it when we submit.